challenge_storing_groceries/src/challenge_storing_groceries/inspection_result_designator.py

```diff
@@ -0,0 +1,16 @@
+from robot_smach_states.util.designators import Designator
+
+
+class InspectionResultDesignator():
```

Comment:
This name suggests the class is a designator while it's not (at least, it does not inherit from robot_skills designator nor does it have the same interface). This might be confusing.